api/authentication.js

```diff
@@ -304,6 +309,19 @@ const init = () => {
           // 1
           // if the user already has a githubProviderId, don't override it
           if (req.user.githubProviderId) {
+            /*
+              Update the cached content of the github profile that we store
+              in redis for the graphql resolver. This allows us to put a button
+              on the client for a user to re-connect a github profile from
+              the web app which will update the cache with any changed usernames
+            */
+            await cache.set(
+              `githubProfile:${req.user.id}`,
```

Comment:
nit: let's key this as `user:${req.user.id}:github`

EDIT: We should probably move this into a shared util in `shared/` so the key is consistent between all invocations even if we change it in the future.

Reply:
Will do!